test_data/pil/sum_via_witness_query.pil

```diff
@@ -7,7 +7,13 @@ namespace Sum(%N);
     pol fixed ISALMOSTLAST(i) { if i == %last_row - 1 { 1 } else { 0 } };
     pol fixed ISFIRST = [ 1, 0 ] + [0]*;
 
-    col witness input(i) query ("input", i);
+    col witness input(i) query match i {
+        // A non-exhaustive match statement is the only way to return "None"
```

Comment:
Is this true? I tried doing something like `if i < 3 { ("input", i) } else { None }`, but there is no symbol `None` and it's also not legal just to leave the `else` branch.